### PR TITLE
Reorganize, add items and add TOC

### PR DESCRIPTION
**Pull request by RachaneeSaeng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,119 +1,182 @@
 # Awesome .NET Thailand 
-## [https://github.com/dotnetthailand/awesome-dotnet-thailand](https://github.com/dotnetthailand/awesome-dotnet-thailand)
```
Comment (aaronamm) [inline README.md:2]: It like redundant but I think it is easy for someone to recognize a URL of repository. LoL

What do you think? Can you confirm that we should remove this?

File: README.md [hunk 2]
```diff
@@ -1,119 +1,182 @@
...
+  * [Cloud Resource Management](#cloud-resource-management)
+  * [CI/CD](#ci-cd)
+- [Others](#others)
+  * [Business Intelligence](#business-intelligence)
+  * [Email Server](#email-server)
+  * [Notification Server](#notification-server)
+  * [Document, Excel](#document--excel)
+
+## Entry Points
+### Entry Points
```
Comment (aaronamm) [inline README.md:49]: Any idea that we can make them different?
I know that you are grouping main/sub-topic.
However, main Entry Points and sub Entry Point look duplicated code and can be a mistake to me.

**Comment by aaronamm** [comment]
Good job, thanks for your contribution after you have fixed my feedback, I will merge your PR.

File: README.md (hunk 1 above)
Comment (RachaneeSaeng) [reply README.md:2]: That's reasonable. I removed it in order to just get rid of it from TOC, but I have found a tool which can skip unwanted part from TOC so don't need to remove this line anymore.
https://ecotrust-canada.github.io/markdown-toc/

File: README.md (hunk 2 above)
Comment (RachaneeSaeng) [reply README.md]: Changed the group name to be `Internet Facing`.

**Pull request merged**